kpi/migrations/0056_fix_asset_permissions.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 3.2.15 on 2024-02-12 15:47
+
+from django.conf import settings
+from django.db import migrations
+
+from kpi.constants import (
+    ASSET_TYPE_SURVEY,
+    PERM_ADD_SUBMISSIONS,
+)
+
+
+def remove_add_submissions_for_anonymous_user(apps, schema_editor):
+
+    Asset = apps.get_model('kpi', 'Asset')  # noqa
+    ObjectPermission = apps.get_model('kpi', 'ObjectPermission')  # noqa
+    Permission = apps.get_model('auth', 'Permission')  # noqa
+    permission_id = Permission.objects.get(codename=PERM_ADD_SUBMISSIONS).pk
+
+    ObjectPermission.objects.filter(
+        deny=False,
+        inherited=False,
```

Comment:
Should we have this restriction in the filter? It shouldn't ever be valid to have `add_submissions` on non-`survey` assets